Azure/sonic-utilities
# Update consutil for picocom v1.7

**- What I did**
Updated consutil for picocom v1.7 by removing QUIET flag and accounting for /var/lock files
**- How I did it**
Removed any code that used QUIET flag. Added code in `clear()` that removes /var/lock/LCK..ttyUSB<linenumber> file
**- How to verify it**
Run `connect line <linenumber>` or `connect device <devicename>` and `clear line <linenumber>` on device with picocom v1.7. Ensure that you can successfully connect to devices, that you get an error for trying to connect to a busy device, and that `clear` successfully clears a busy line.

-->
